config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -185,18 +185,22 @@ public void run() {
                     setFailResult(configFuture);
                     return;
                 }
-                if (configFuture.getOperation() == ConfigOperation.GET) {
-                    String result = CONFIG.getString(configFuture.getDataId());
-                    configFuture.setResult(result == null ? configFuture.getContent() : result);
-                } else if (configFuture.getOperation() == ConfigOperation.PUT) {
-                    //todo
-                    configFuture.setResult(Boolean.TRUE);
-                } else if (configFuture.getOperation() == ConfigOperation.PUTIFABSENT) {
-                    //todo
-                    configFuture.setResult(Boolean.TRUE);
-                } else if (configFuture.getOperation() == ConfigOperation.REMOVE) {
-                    //todo
-                    configFuture.setResult(Boolean.TRUE);
+                try {
+                    if (configFuture.getOperation() == ConfigOperation.GET) {
+                        String result = CONFIG.getString(configFuture.getDataId());
+                        configFuture.setResult(result);
+                    } else if (configFuture.getOperation() == ConfigOperation.PUT) {
+                        //todo
+                        configFuture.setResult(Boolean.TRUE);
+                    } else if (configFuture.getOperation() == ConfigOperation.PUTIFABSENT) {
+                        //todo
+                        configFuture.setResult(Boolean.TRUE);
+                    } else if (configFuture.getOperation() == ConfigOperation.REMOVE) {
+                        //todo
+                        configFuture.setResult(Boolean.TRUE);
+                    }
+                } catch (Exception e){
+                    setFailResult(configFuture);
```

Comment:
logger.error

Reply:
it would be like this.

`2019-04-29 20:55:15.578 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property store.flush-disk-mode, try to use default value instead.
2019-04-29 20:55:16.117 INFO [main]io.seata.core.rpc.netty.AbstractRpcRemotingServer.start:152 -Server started ... 
2019-04-29 20:55:16.136 INFO [main]io.seata.common.loader.EnhancedServiceLoader.loadFile:192 -load RegistryProvider[Redis] extension by class[io.seata.discovery.registry.redis.RedisRegistryProvider]
2019-04-29 20:55:16.138 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.cluster, try to use default value instead.
2019-04-29 20:55:16.139 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.password, try to use default value instead.
2019-04-29 20:55:16.141 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.test.on.borrow, try to use default value instead.
2019-04-29 20:55:16.141 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.test.on.return, try to use default value instead.
2019-04-29 20:55:16.142 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.test.while.idle, try to use default value instead.
2019-04-29 20:55:16.142 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.max.idle, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.min.idle, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.max.active, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.max.total, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.timeout, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.max.wait, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.num.tests.per.eviction.run, try to use default value instead.
2019-04-29 20:55:16.143 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.time.between.eviction.runs.millis, try to use default value instead.
2019-04-29 20:55:16.144 WARN [configOperate_1_2]io.seata.config.FileConfiguration.run:204 -Could not found property registry.redis.min.evictable.idle.time.millis, try to use default value instead.`